internal/highlight/syntect_language_map.go

```diff
@@ -345,4 +345,5 @@ var SyntectLanguageMap = map[string]string{
 	"yaml":                 "yaml",
 	"yml":                  "yml",
 	"sublime-syntax":       "sublime-syntax",
+	"kotlin":               "kt",
```

Comment:
I think you also want the other extensions I mentioned here.
```suggestion
	"kotlin":               "kt",
	"kt":                     "kt",
	"kts":                   "kts",
	"jetl":                   "jetl",
	"jet":                    "jet",
```

Otherwise I think some search result types won't get highlighted for these languages.

Reply:
(my formatting may be off above, best to apply that in your editor)

Reply:
jet/jetl are considered deprecated according to [here](https://discuss.kotlinlang.org/t/whats-the-deference-between-kt-and-kts-files-jet-and-jetl-files/1058), should we include them anyways? Ive never seen them in the wild and the only results for such on dotcom includes either binary files or java source code to-be template processed

Reply:
The syntax definition has them, so I'd be inclined to have them here too - but I'm happy either way! :) `kt` and `kts` seem important, though, no?

Reply:
Those two we can add for sure yea :)